api/src/main/java/org/apache/iceberg/expressions/UnboundPredicate.java

```diff
@@ -284,6 +285,12 @@ public String toString() {
         return term() + " startsWith \"" + literal() + "\"";
       case NOT_STARTS_WITH:
         return term() + " notStartsWith \"" + literal() + "\"";
+      case ST_INTERSECTS:
+      case ST_DISJOINT:
+        {
+          String functionName = op().name().toLowerCase(Locale.ROOT);
```

Comment:
Since both will be one line now, there is no benefit to combine the two operations. we can hardcode the function name in the string construction, which is also consistent with other cases.